app/controllers/concerns/atmosphere/api/application_controller_ext.rb

```diff
@@ -25,6 +29,15 @@ def token
           request.headers[Devise::Strategies::TokenAuthenticatable.header_key].
             presence
       end
+
+      def set_project
+        current_user.project = project if current_user
+      end
+
+      def project
+        params[Air.config.project_key] ||
+          request.headers[Air.config.header_project_key]
```

Comment:
Will this be returned by the MI (if so -- when?) or is this just a preparatory measure on our part?